### PR TITLE
Added Unit tests to eos_logging

### PR DESCRIPTION
**Pull request by GomathiselviS**
SUMMARY

Added Unit Testcases to eos_logging module

ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
test/units/modules/network/eos/test_eos_logging.py

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

File: test/units/modules/network/eos/test_eos_logging.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        self.mock_get_config.stop()
+        self.mock_load_config.stop()
+
+    def load_fixtures(self, commands=None, transport='cli'):
+        self.get_config.return_value = load_fixture('eos_logging_config.cfg')
+        self.load_config.return_value = dict(diff=None, session='session')
+
+    def test_eos_setup_host_logging_idempotenet(self):
+        set_module_args(dict(dest='host', name='175.16.0.10', state='present'))
+        commands = ['logging host 175.16.0.10']
```
Comment (Qalthos) [inline test/units/modules/network/eos/test_eos_logging.py]:
```suggestion
```

File: test/units/modules/network/eos/test_eos_logging.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        self.execute_module(changed=False, commands=[])
+
+    def test_eos_setup_host_logging(self):
+        set_module_args(dict(dest='host', name='175.16.0.1', state='present'))
+        commands = ['logging host 175.16.0.1']
+        self.execute_module(changed=True, commands=commands)
+
+    def test_eos_buffer_size_outofrange(self):
+        set_module_args(dict(dest='buffered', size=5))
+        commands = ['logging buffered 5']
```
Comment (Qalthos) [inline test/units/modules/network/eos/test_eos_logging.py]:
```suggestion
```

**Review by Qalthos — CHANGES_REQUESTED** [review]
Again with commands that aren't asserted. You could probably at least do something with `load_config.assert_called_once_with(...)` if you really wanted to, but that's probably overkill

**Comment by GomathiselviS** [comment]
recheck

File: test/units/modules/network/eos/test_eos_logging.py (hunk 2 above)
Comment (GomathiselviS) [reply test/units/modules/network/eos/test_eos_logging.py]: Done!

**Review by Qalthos — APPROVED** [review]

**Pull request merged**